test/sanity_test.sh

```diff
@@ -14,6 +20,31 @@ zip_result=-1
 sec_result=-1
 hpre_result=-1
 
+TEST_FILE=test_uadk_lib.c
+
+# failed: return 1; success: return 0
+check_uadk_lib()
+{
+	# wd_get_acce_list() is available in v2. Use this function to test
+	# whether libwd.so is v2.
+	echo "#include <stdio.h>" > $TEST_FILE
```

Comment:
instead of using several echos, use a heredocument:

cat << EOF > ${TEST_FILE}
#include ...
EOF

Reply:
Fixed.